### PR TITLE
few improvements

### PR DESCRIPTION
**Pull request by OlekAleksander**

### CODE REVIEW

**Comment by camwhi-te** [comment]
As far as I can see, the admin area improvement does not include the link to the Discord.

**Comment by OlekAleksander** [comment]
sorry i forgot, fixing

**Comment by OlekAleksander** [comment]
is it better now?

File: resources/scripts/components/dashboard/activity/ActivityLogContainer.tsx [hunk 1]
```diff
@@ -35,6 +35,8 @@ export default () => {
 
     return (
         <PageContentBlock title={'Account Activity Log'}>
+            <h1 css={tw`text-5xl`}>Account activity log</h1>
+            <h3 css={tw`text-2xl text-neutral-500`}>View your recent Activity</h3>
```
Comment (camwhi-te) [inline resources/scripts/components/dashboard/activity/ActivityLogContainer.tsx:40]: Would be better to change this to:

Account Activity
View your recent activity.

File: resources/views/admin/index.blade.php [hunk 2]
```diff
@@ -32,27 +32,26 @@
...
+                    Your panel is <strong>not up-to-date!</strong> The latest version is <a href="https://github.com/jexactyl/jexactyl/releases/v{{ $version->getPanel() }}" target="_blank"><code>{{ $version->getPanel() }}</code></a> and you are currently running version <code>{{ config('app.version') }}</code>.
                 @endif
             </div>
         </div>
     </div>
 </div>
 <div class="row">
-    <div class="col-xs-6 col-sm-3 text-center">
-        <a href="{{ $version->getDiscord() }}"><button class="btn btn-warning" style="width:100%;"><i class="fa fa-fw fa-support"></i> Get Help <small>(via Discord)</small></button></a>
+    <div class="col-xs-6 col-sm-3 text-center"><a href="https://discord.gg/ECYh2DFNkF"><button class="btn btn-warning" style="width:100%;"><i class="fa fa-fw fa-support"></i> Discord </button></a>
```
Comment (camwhi-te) [inline resources/views/admin/index.blade.php]: Please set this link to: https://discord.gg/qttGR4Z5Pk

**Review by camwhi-te — CHANGES_REQUESTED** [review]
Waiting for changes to be made as per my comnments.

**Comment by OlekAleksander** [comment]
Done

**Pull request merged**